src/Sylius/Behat/Context/Api/Admin/ManagingPromotionsContext.php

```diff
@@ -120,7 +120,7 @@ public function iMakeItCouponBased(): void
      * @When I set its priority to :priority
      * @When I remove its priority
      */
-    public function iRemoveItsPriority(?int $priority = null): void
+    public function iRemoveItsPriority(int $priority = -1): void
```

Comment:
Why? I believe `null` was added because of the `I remove its priority` step. But reading this fragment, I have no idea why we set `-1`.

Reply:
Yes, you are right. I restored the default null value and added a new scenario where the value -1 is indirectly passed.
`Scenario: Setting promotion to the lowest priority`